app/views/locations/index.html.erb

```diff
@@ -8,20 +8,22 @@
       <th>City, State</th>
       <th>Zip</th>
       <th>County</th>
-      <th>ServiceArea</th>
+      <th>Region</th>
+      <th>Neighborhood</th>
```

Comment:
Def want to revisit Region/Neighborhood stuff on Location. Not sure if this will ultimately get supplanted by nested ServiceAreas ...